src/lualib/ArraySplice.ts

```diff
@@ -0,0 +1,93 @@
+// https://www.ecma-international.org/publications/files/ECMA-ST/Ecma-262.pdf
+function __TS__ArraySplice<T>(list: T[], start: number, deleteCount: number, ...items: T[]): T[] {
+    // 1. 2.
```

Comment:
I would either write out the pseudo-code in the comments or just remove them. These numbers don't really add anything imo.